back/engines/commercial/analytics/spec/models/analytics/fact_visit_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Analytics::FactVisit, type: :model do
+  it 'Can create a visit' do
+    create(:fact_visit)
+  end
+
+  it 'Can associate a visit with multiple projects in the dimension_projects db view' do
+    project1 = create(:project)
+    project2 = create(:project)
+    dimension_project1 = Analytics::DimensionProject.first
+    dimension_project2 = Analytics::DimensionProject.last
+
+    visit = create(:fact_visit)
+    visit.dimension_projects << dimension_project1
+    visit.dimension_projects << dimension_project2
+
+    assert(visit.dimension_projects[0].id == project1.id)
```

Comment:
We don't use `assert` (`assert` is defined by MiniTest, while we use Rspec).

Reply:
How strange that this seems to work then! Will change

Reply:
It works because it comes with Rails, but we stick to Rspec :)